### PR TITLE
DAO layer is created, FamilyDto is added to Human class, getAllFamili…   

### PR DESCRIPTION
**Pull request by 2kfarid3**
…es and createNewFamily method is written,HappyFamilyApp class is written temporary

### CODE REVIEW

File: src/main/java/az/edu/strangers/Human.java [hunk 1]
```diff
@@ -11,6 +13,7 @@ public class Human {
     private Integer IQ;
     private Map<DayOfWeek, String> schedule;
     private Family family;
+    private FamilyDto familyDto;
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/strangers/Human.java]: this model should not be here

File: src/main/java/az/edu/strangers/controller/FamilyController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package az.edu.strangers.controller;
+
+import az.edu.strangers.dto.FamilyDto;
+import az.edu.strangers.dto.ManDto;
+import az.edu.strangers.dto.WomanDto;
+import az.edu.strangers.service.FamilyService;
+
+public class FamilyController {
+
+    FamilyService familyService;
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/strangers/controller/FamilyController.java]: make private

File: src/main/java/az/edu/strangers/dao/FamilyDao.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package az.edu.strangers.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface FamilyDao<T> {
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/strangers/dao/FamilyDao.java]: I think no need Generic here, this dao for the `Family` class

File: src/main/java/az/edu/strangers/service/FamilyServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+    @Override
+    public Long countFamiliesWithMemberNumber(Integer number) {
+        return null;
+    }
+
+    @Override
+    public FamilyDto createNewFamily(ManDto man, WomanDto women) {
+        Woman mother = new Woman(women.getName(), women.getSurname(), women.getYear());
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/strangers/service/FamilyServiceImpl.java]: why recreate sended objects?

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

**Comment by ayten-github** [comment]
Because of some changes

**Comment by ayten-github** [comment]
.

**Pull request closed without merging**